fedimint-atomic-broadcast/src/db.rs

```diff
@@ -2,27 +2,31 @@ use std::io::Cursor;
 
 use fedimint_core::db::Database;
 use fedimint_core::encoding::{Decodable, Encodable};
-use fedimint_core::impl_db_record;
+use fedimint_core::{impl_db_lookup, impl_db_record};
 
 use crate::SignedBlock;
 
 #[derive(Debug, Encodable, Decodable)]
-struct SignedBlockKey(u64);
+pub struct SignedBlockKey(pub u64);
+
+#[derive(Debug, Encodable, Decodable)]
+pub struct SignedBlockPrefix;
 
 impl_db_record!(
     key = SignedBlockKey,
     value = SignedBlock,
-    db_prefix = 0x00,
+    db_prefix = 0x04,
```

Comment:
elsewhere in the codebase we use enums for these constants

Reply:
We will switch to the whims before we merge, will require to move the fedimint-atomic-broadcast crate in a module in fedimint-server to avoid a cyclical dependency between the crates